lookbook/docs/patterns/10-page-headers.md.erb

```diff
@@ -0,0 +1,67 @@
+This component is an adaptation for OpenProject requirements of the React component `PageHeader` of Primer. This document will only state the points that differ from what **[Primer specifies](https://primer.style/components/page-header)** about this component in their documentation.
+
+## Anatomy
+
+There are minor adjustments on how the composition of this component is defined in Primer for OpenProject:
+
+**[Context bar](https://primer.style/components/page-header#context-bar):** Will be always present in all pages and will change depending on the viewport:
+
+- **Desktop:** The content of the context bar will be a [breadcrumb](https://primer.style/components/breadcrumbs).
+- **Tablet:** The content of the context bar will be a [breadcrumb](https://primer.style/components/breadcrumbs).
+- **Mobile:**
+  - The content of the breadcrumb will collapse to a single argument using the parent link style.
```

Comment:
Could the word "argument" here be replaced with "element" so that it reads "will collapse to a single element using the parent link style"? This makes it easier for non-devs to understand it too.